data/publiccloud/terraform/azure.tf

```diff
@@ -106,35 +106,35 @@ resource "random_id" "service" {
 
 
 resource "azurerm_resource_group" "openqa-group" {
-  name     = "${var.name}-${element(random_id.service.*.hex, 0)}"
+  name     = "${var.name}-${element(random_id.service[*].hex, 0)}"
   location = var.region
 
   tags = merge({
     openqa_created_by   = var.name
     openqa_created_date = timestamp()
-    openqa_created_id   = element(random_id.service.*.hex, 0)
+    openqa_created_id   = element(random_id.service[*].hex, 0)
   }, var.tags)
 }
 
 resource "azurerm_public_ip" "openqa-publicip" {
-  name                = "${var.name}-${element(random_id.service.*.hex, count.index)}-public-ip"
+  name                = "${var.name}-${element(random_id.service[*].hex, count.index)}-public-ip"
   location            = var.region
   resource_group_name = azurerm_resource_group.openqa-group.name
-  allocation_method   = "Dynamic"
+  allocation_method   = "Static"
```

Comment:
can you elaborate need of this change ? 
I see two risks here : 
- it might be more expensive 
- it might create issues on cleanup 
:thinking:

Reply:
In September 2025, the "Basic" tier Public IP resource [will get retired](https://azure.microsoft.com/en-us/updates?id=upgrade-to-standard-sku-public-ip-addresses-in-azure-by-30-september-2025-basic-sku-will-be-retired) and as far as I've found out, the only other tier is "Standard" and that one doesn't support "Dynamic" allocation method, so here we are.

Reply:
https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-basic-upgrade-guidance#basic-sku-vs-standard-sku

Reply:
ok , make sense than